### PR TITLE
Add an example filter and http check

### PR DESCRIPTION
**Pull request by asachs01**
In the interest of being able to have folks follow along with this demo, I've added an http check for them to modify, as well as a filter so that they won't get pinged to death with Slack messages.

### CODE REVIEW

**Comment by asachs01** [comment]
@calebhailey @mattbeninghof @portertech y'all cool if I pull this in?

File: go/config/filters/filter_interval_60_hourly.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "spec": {
+    "action": "allow",
+    "expressions": [
+      "event.check.interval == 60",
+      "event.check.occurrences == 1 || event.check.occurrences % 60 == 0"
+    ],
+    "runtime_assets": []
+  }
+}
```
Comment (calebhailey) [inline go/config/filters/filter_interval_60_hourly.json]: Is there a reason why this one manifest is json instead of yaml?

Comment (asachs01) [reply go/config/filters/filter_interval_60_hourly.json]: Ah, I should just remove it.

Comment (calebhailey) [reply go/config/filters/filter_interval_60_hourly.json]: :+1:

**Pull request merged**